app/code/Magento/Ui/view/base/web/js/grid/url-filter-applier.js

```diff
@@ -12,11 +12,15 @@ define([
     return Component.extend({
         defaults: {
             listingNamespace: null,
+            selectComponentName: null,
```

Comment:
The filter may not be a select, please adjust the logic accordingly
```suggestion
            filterComponentName: null,
```